enhancements/microshift/microshift-default-csi-plugin.md

```diff
@@ -0,0 +1,419 @@
+---
+title: microshift-default-csi-plugin
+authors:
+  - copejon
+reviewers:
+  - fzdarsky # Frank Zdarsky
+  - oglok # Ricky de'Noriega
+  - mangelajo #  Miguel Angel Ajo Pelayo
+  - nbalacha # Nithya Balachandran
```

Comment:
Please tag @openshift/storage for all storage related enhancements. I don't see anything wrong in this enhancement (good work!), but we'd like to know what's coming to OCP.